src/base/bittorrent/session.cpp

```diff
@@ -441,6 +441,8 @@ Session::Session(QObject *parent)
 #if defined(Q_OS_WIN)
     , m_OSMemoryPriority(BITTORRENT_KEY("OSMemoryPriority"), OSMemoryPriority::BelowNormal)
 #endif
+    , m_isReannounceWhenAddressChanged(BITTORRENT_SESSION_KEY("ReannounceWhenAddressChanged"), false)
+    , m_lastExternalIP()
```

Comment:
This line is redundant, please remove it.

Reply:
Removed, and no warning shows in msvc 2019.